src/test/java/comtest/LoginTest.java

```diff
@@ -1,4 +1,4 @@
-package com;
+package comtest;
```

Comment:
sao phai rename folder com thanh comtest the : ))

Reply:
này hôm qua em clone về, lúc run nó kêu bị trùng cái com base
nên k run dc a, nên e fix tạm thành comtest mới run dc:))